web/src/api/manager.ts

```diff
@@ -42,6 +42,6 @@ const finishInstallation = () => post("/api/manager/finish");
 /**
  * Returns the binary content of the YaST logs file.
  */
-const fetchLogs = () => get("/api/manager/logs.tar.gz");
+const fetchLogs = () => get("/api/manager/logs/store");
```

Comment:
Have you tried this change? Removing the `.tar.gz` will result in a `.gz` file that will not open when clicked on.

Reply:
i used wrong content type header ... should be fixed now